draft-ietf-quic-transport.md

```diff
@@ -2736,7 +2736,7 @@ available.
 
 ## Protected Packets {#packet-protected}
 
-All QUIC packets except Version Negotiation and Retry packets use authenticated
+All QUIC packets except Version Negotiation packets use authenticated
 encryption with additional data (AEAD) {{!RFC5116}} to provide confidentiality
```

Comment:
The confidentiality bit means that this statement still applies to Retry.

Reply:
I'm not sure whether Retry packets have less confidentiality protection than Initial packets?

Reply:
They're not encrypted, so kind of by definition....